Drive/rover_drive_system.hpp

```diff
@@ -10,6 +10,15 @@
 #include "../Common/esp.hpp"
 #include "wheel.hpp"
 
+#include "../../../library/devices/sensors/battery/max17043.hpp"
+// #include "../../../libray  "
+#include "../../../library/peripherals/lpc17xx/i2c.hpp"
+//#include "../../../library/lpc40xx/peripherals/gpio.hpp"
+#include "peripherals/lpc40xx/gpio.hpp"
+#include "../../../library/peripherals/hardware_counter.hpp"
+#include "../../../library/devices/sensors/battery/ltc4150.hpp" 
+
+
```

Comment:
Remove the ../../../ from the include statements. This is just a vs code bug. The code should compile without them if dev2 was setup properly.